### ISSUE TITLE
[APP SUBMITTED]: DelugeD: Unable to set priority for Torrent

### ISSUE THREAD

**Issue by phraymond** [pymedusa/Medusa#2618]
### INFO
**Python Version**: `2.7.13 (default, Dec 22 2016, 09:22:15) [GCC 6.2.1 20160822]`
**Operating System**: `Linux-4.9.19-unRAID-x86_64-with`
**Locale**: `None`
**Branch**: [master](../tree/master)
**Database**: `44.4`
**Commit**: pymedusa/Medusa@a0116154f262786d6f5274459cb8f9cdb23012cc
**Link to Log**: https://gist.github.com/1a514fbe3cfb218a4c981ee85b3ddd0f
### ERROR
<pre>
2017-04-20 10:06:37 ERROR    SEARCHQUEUE-BACKLOG-70047 :: [a011615] DelugeD: Unable to set priority for Torrent
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by fernandog** [comment]
@phraymond 
probably it couldn't connect to Deluge
you should disable tornado logs in general config

**Comment by fernandog** [comment]
no reply

**Comment by phraymond** [comment]
No reply as I hadn't realized the app had submitted such a slew of issues! I was looking for an issue I'd opened, clicked on my profile and was extremely startled to see two pages of "App Submitted" issues! I assume this was from my first attempt to move from Sickrage to Medusa, when it threw a slew of warnings at me and I clicked "Submit". Only one github page popped up, which I then filled out. I never saw the rest. 
Bug, perhaps?

**Comment by phraymond** [comment]
I took a closer look and 2645 is the one that was displayed after I clicked "Submit". Being the only one that I saw after clicking, I proceeded to fill it out with the issue I was experiencing, again not realizing that the app had opened the others. I guess that explains why all the other errors went away, as you went through and resolved them without me even realizing! Nicely done! Responsiveness was the primary reason I decided to move to Medusa from SR. Thank you!
